tests/Console/ConfigurationResolverTest.php

```diff
@@ -986,12 +986,12 @@ public function testResolveRulesWithUnknownRules()
     {
         $this->setExpectedException(
             \PhpCsFixer\ConfigurationException\InvalidConfigurationException::class,
-            'The rules contain unknown fixers (bar).'
+            'The rules contain unknown fixers ("bar", "binary_operator_space" maybe use "binary_operator_spaces").'
```

Comment:
Maybe:
```diff
-'The rules contain unknown fixers ("bar", "binary_operator_space" maybe use "binary_operator_spaces", "baz").'
+'The rules contain unknown fixers ("bar", "binary_operator_space" (did you mean "binary_operator_spaces"?), "baz").'
```

Reply:
I played around with the copy and picked the current one as it is more compact when multiple unknown rules are used. However your suggestion looks better for when one or two unknown rules are passed.

Reply:
for me it was like:
>  "xxx" maybe use "yyy"

no! I want to use XXX, why would I need to use YYY?

> "xxx" (did you mean "yyy"?)

oh, I made a typo !

Reply:
:+1: for @keradus' suggestion, I would even remove the main parentheses:
```diff
-The rules contain unknown fixers ("bar", "binary_operator_space" maybe use "binary_operator_spaces", "baz").
+The rules contain unknown fixers: "bar", "binary_operator_space" (did you mean "binary_operator_spaces"?), "baz".
```